usaspending_api/common/query_with_filters.py

```diff
@@ -105,20 +105,39 @@ class _TimePeriods(_Filter):
 
     @classmethod
     def generate_elasticsearch_query(cls, filter_values: List[dict], query_type: _QueryType, **options) -> ES_Q:
+
+        # Temporary until rest of dependencies are updated
+        if "time_period_obj" not in options or options.get("time_period_obj") is None:
+            return cls._default_elasticsearch_query(filter_values, query_type, **options)
+
         time_period_query = []
-        if options:
-            gte_field = options.get("gte_field", "action_date")
-            lte_field = options.get("lte_field", "date_signed" if query_type == _QueryType.AWARDS else "action_date")
+        for v in filter_values:
+            time_period_obj = options["time_period_obj"]
+            time_period_obj.filter_value = v
```

Comment:
I understand single letter vars for simple looping through items in list comprehensions but here it was a little confusing seeing line 116. Could this var be renamed to `filter_value`? I know this wasn't originally your code but I would still like it fixed for clarity. 

Also applies to the loops starting at 40, 97, 134, and 159.

Reply:
Addressed